complete/helpers.c

```diff
@@ -45,6 +44,7 @@
 #include "menu/lib.h"
 #include "compapi.h"
 #include "data.h"
+#include "mutt/ctype2.h"
```

Comment:
You should include this from `mutt/lib.h`. Most files would already include `mutt/lib.h` so you don't have to tweak the includes of everything